torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -2011,11 +2025,20 @@ def load_custom_op_wrapper(self):
 if (codecache_module.get() == NULL) {
     throw std::runtime_error("Failed to load torch._inductor.codecache");
 }
-RAIIPyObject custom_op_wrapper(PyObject_GetAttrString(codecache_module, "custom_op_wrapper"));
-if (custom_op_wrapper.get() == NULL) {
+custom_op_wrapper_ptr = std::make_unique<RAIIPyObject>(PyObject_GetAttrString(codecache_module, "custom_op_wrapper"));
+if (custom_op_wrapper_ptr.get()->get() == NULL) {
     throw std::runtime_error("Failed to load torch._inductor.codecache.custom_op_wrapper");
 }"""
-        self.writelines(lines.split("\n"))
+
+        # Use ptr here because RAIIPyObject does not have a default constructor
+        declarations_before_scope = [
+            "std::unique_ptr<RAIIPyObject> custom_op_wrapper_ptr;"
```

Comment:
Why not add a default constructor and assignment operator? Allocating a pointer in the heap and only using it locally seems wasteful.